src/main/java/Duke.java

```diff
@@ -1,19 +1,49 @@
 import java.util.Scanner;
 
 public class Duke {
+    public static final int SKIP_DEADLINE_NUMBER = 9;
+    public static final int SKIP_EVENT_NUMBER = 6;
+    public static final int SKIP_DONE_NUMBER = 5;
+    public static final int SKIP_SEPARATION_NUMBER = 4;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
-
         Duke.greet();
         Scanner in = new Scanner(System.in);
         String command = in.nextLine();
+        String description;
+        String time;
+        Task[] list = new Task[100];
+        int separatePosition;
+        int number = 0;
         while (!command.equals("bye")) {
-            Duke.echo(command);
+            if (command.equals("list")) {
+                Duke.displayList(list, number);
+            } else if (command.startsWith("done")) {
+                Duke.done(list, command, number);
+            } else if (command.startsWith("todo")) {
+                list[number] = new ToDo(command.substring(5));
+                number = Duke.add(list[number], number);
+            } else if (command.startsWith("deadline")) {
+                separatePosition = command.indexOf('/');
+                description = command.substring(SKIP_DEADLINE_NUMBER, separatePosition-1);
+                time = command.substring(separatePosition+SKIP_SEPARATION_NUMBER);
+                list[number] = new Deadline(description, time);
+                number = Duke.add(list[number], number);
+            } else if (command.startsWith("event")) {
+                separatePosition = command.indexOf('/');
+                description = command.substring(SKIP_EVENT_NUMBER, separatePosition-1);
+                time = command.substring(separatePosition+SKIP_SEPARATION_NUMBER);
+                list[number] = new Event(description, time);
+                number = Duke.add(list[number], number);
+            } else {
```

Comment:
if-else used correctly, but a switch statement might be clearer in showing the consequence of different inputs